client/src/pages/DeckBuilder/DeckBuilder.tsx

```diff
@@ -290,6 +301,20 @@ export const DeckBuilder = (props: DeckBuilderProps) => {
                         backgroundColor: `${card.colors[0]}`,
                       }}
                     ></div>
+                    <div
+                      style={{
+                        fontSize: '3vh'
+                      }}
+                    >
+                      <i 
+                        className={`${face === card.id ? "fas fa-star" : "far fa-star"}`}
```

Comment:
can reduce duplicate code 
```suggestion
                        className={`${face === card.id ? 'fas'  : `far`}  fa-star`}
```

Reply:
also prefer `'` over `"`